graphql/schema.js

```diff
@@ -0,0 +1,34 @@
+const typeDefs = `#graphql
+
+  type Book {
+    title: String
+    author: String
+    description: String
+  }
+
+  type Query {
+    books: [Book]
+  }
+`;
+
+const books = [
+    {
+        title: 'The Awakening',
+        description: 'The Awakening is a novel by Kate Chopin published in 1899. It is the story of a woman named Edna Pontellier who is dissatisfied with her life as a wife and mother. She begins to question her traditional beliefs and values and starts to explore her own identity and desires.',
+        author: 'Kate Chopin',
+    },
+    {
+        title: 'City of Glass',
+        description: 'City of Glass is a novel by Paul Auster published in 1989. It is the first book in the "New York Trilogy" and follows the story of three strangers who become entangled in a mystery involving a missing person and a secret society.',
+        author: 'Paul Auster',
+    },
+];
+
+const resolvers = {
+    Query: {
+        books: () => books,
```

Comment:
should this call the db model?

Reply:
This file was just for the initial tests to check if the graphql system works. Right now it is in no way linked to the subgraph in any way. It doesn't call the database model, there is an object defined above it. I was going to write the test cases so I added that.